### PR TITLE
40K 8 more + 2 tokens

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/winged_hive_tyrant.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Winged Hive Tyrant
+ManaCost:3 U R
+Types:Creature Tyranid
+PT:4/4
+K:Flying
+K:Haste
+S:Mode$ Continuous | Affected$ Creature.YouCtrl+HasCounters | AddKeyword$ Flying & Haste | Description$ The Will of the Hive Mind — Other creatures you control with counters on them have flying and haste.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/winged_hive_tyrant.txt:7]: Other missing

File: forge-gui/res/cardsfolder/upcoming/lictor.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Name:Lictor
+ManaCost:3 G
+Types:Creature Tyranid
+PT:3/3
+K:Flash
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | CheckSVar$ X | SVarCompare$ GE1 | TriggerDescription$ Pheromone Trail — When CARDNAME enters the battlefield, if a creature entered the battlefield under an opponent's control this turn, create a 3/3 green Tyranid Warrior creature token with trample.
+SVar:TrigToken:DB$ Token | TokenScript$ g_3_3_tyranid_warrior | CheckSVar$ X | SVarCompare$ GE1
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/lictor.txt:7]: remove svar check, you already did it in line 6 and these params wouldn't do work here anyway

File: forge-gui/res/cardsfolder/upcoming/lictor.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+SVar:X:Count$ThisTurnEntered_Battlefield_Creature.OppCtrl
+DeckHas:Ability$Token
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/lictor.txt:9]: & Type$Warrior

File: forge-gui/res/cardsfolder/upcoming/malanthrope.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+ManaCost:1 G U
+Types:Creature Tyranid
+PT:2/2
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ Scavenge the Dead — When CARDNAME enters the battlefield, exile target player's graveyard. Put a +1/+1 counter on CARDNAME for each creature card exiled this way.
+SVar:TrigExile:DB$ ChangeZoneAll | ValidTgts$ Player | TgtPrompt$ Select target player | Origin$ Graveyard | Destination$ Exile | ChangeType$ Card | SubAbility$ DBCounter | RememberChanged$ True
+SVar:DBCounter:DB$ PutCounter | Defined$ Self | CounterType$ P1P1 | CounterNum$ X | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:X:Remembered$Valid Creature
+DeckHas:Ability$Counter
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/malanthrope.txt:11]: Counter**s**

File: forge-gui/res/cardsfolder/upcoming/tyranid_invasion.txt [hunk 5]
```diff
@@ -0,0 +1,7 @@
+Name:Tyranid Invasion
+ManaCost:3 G
+Types:Sorcery
+A:SP$ Token | TokenOwner$ You | TokenAmount$ X | TokenScript$ g_3_3_tyranid_warrior
+SVar:X:PlayerCountOpponents$Amount
+DeckHas:Ability$Token
+Oracle:Create a number of 3/3 green Tyranid Warrior creature tokens with trample equal to the number of opponents you have.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/tyranid_invasion.txt:7]: AI:RemoveDeck:Random – only good in multiplayer, we don't have a better AI hint yet

File: forge-gui/res/cardsfolder/upcoming/tyranid_prime.txt [hunk 6]
```diff
@@ -0,0 +1,8 @@
+Name:Tyranid Prime
+ManaCost:1 G U
+Types:Creature Tyranid
+PT:0/4
+K:Evolve
+S:Mode$ Continuous | Affected$ Creature.Other+YouCtrl | AddKeyword$ Evolve | Description$ Synapse Creature — Other creatures you control have evolve.
+DeckHas:Ability$Counter & Keyword$Evolve
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/tyranid_prime.txt:7]: Counter**s**

**Review by tool4ever — APPROVED** [review]

**Pull request merged**